src/providers/messages.js

```diff
@@ -84,3 +85,7 @@ function isAlarmType(type) {
 function isCommandType(type) {
     return type === OBJECT_TYPES.COMMANDS_OBJECT_TYPE;
 }
+
+function isMdbChangesType(type) {
+    return type === 'yamcs.mdbchanges';
```

Comment:
You can swap this out for the `OBJEC_TYPES.MDB_CHANGES` once you add it to "const.js"

Reply:
Done